geoschem.go

```diff
@@ -114,7 +118,15 @@ type GEOSChem struct {
 // 'IJ_AVG_S_xxx'.
 //
 // If msgChan is not nil, status messages will be sent to it.
-func NewGEOSChem(GEOSA1, GEOSA3Cld, GEOSA3Dyn, GEOSI3, GEOSA3MstE, GEOSApBp, GEOSChemOut, VegTypeGlobal, startDate, endDate string, dash bool, msgChan chan string) (*GEOSChem, error) {
+//
+// chemRecordInterval is the time interval between different records in
+// the CTM output, as specified by the user.
+//
+// chemFileInterval is the time interval of each file, as specified by
+// the user.
+//
+// If noHour is true, then the filename does not include the time in it.
```

Comment:
If nHour is true, then the GEOS-Chem output files will be assumed to not contain a time time dimension.